### ISSUE TITLE
Not able to create new option in model Quick Test - missing parametertype idd

### ISSUE THREAD

**Issue by samatrhea** [RHEAGROUP/CDP4-WebServices-Community-Edition#6]
This bug report was originally reported here https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues/14. This seems to be a server bug and not a CDP4-IME bug

[20180508 CDP4IME - Not able to create option due to missing parameter type iid.log](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/files/1984269/20180508.CDP4IME.-.Not.able.to.create.option.due.to.missing.parameter.type.iid.log)
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have verified that I am running the latest version of the CDP4-IME
- [X] I have searched [open](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

On the demo server, small model "Quick Test" is available, with limited set of EDs and parameters. On this model, there are a few parameters with option dependency. One option available, the default option.
Trying to create a new option: this gives an error message, no new option created.

### Steps to Reproduce

Open "Quick Test" model on the demo server and try to create a new option.

### System Configuration

- CDP4 version:
  - [X] CDP4-IME: 1.0.8
  - [ ] Other: 
  -
- Web Services:
  - [x] CDP4 Web Services version: 0.13.0
  - [x] CDP4Common: 1.0.6
- Environment (Operating system, version and so on):
  - [ ] Microsoft Excel (32/64 bit): 
  - [ ] Windows OS (32/64 bit): 
  - [ ] Additional information:

**Comment by samatrhea** [comment]
This may be occurring due to the use of the `IParameterTypeService` in the `OptionSideEffect`, we should start the investigations there. @naron-rhea what do you think?

**Comment by naron-rhea** [comment]
IParameterTypeService is a service for the abstract type ParameterType but I think the permission does exist for this type.

In any case I have tried to download the data from the test-server so that I could create an option in the model but I did succeed in performing this operation (with an existing option-dependent parameter).

At the moment I am unsure what the issue is as I cannot reproduce the problem.
